guide/content/components/start-button.slim

```diff
@@ -12,4 +12,8 @@ p
   caption: "Start button",
   code: start_button_normal)
 
+== render('/partials/example.*',
+  caption: "Start button as a submit button",
+  code: start_button_as_button)
```

Comment:
I think we could benefit from a sentence here on why one might want to use a form/button here rather than just a link.

Not sold on this but as a starting off point

```suggestion
  code: start_button_as_button) do

  markdown:
    If you need to submit data you can make the component render a form
    containing a button instead of a link by passing `as_button: true`. This
    approach uses Rails' `button_to` helper and will render a form that will
    `POST` to the target URL.

  div.govuk-warning-text
    span.govuk-warning-text__icon aria-hidden="true"
      | !
    strong.govuk-warning-text__text
      span.govuk-warning-text__assistive Warning
      | The GOV.UK Design system
      =< link_to("advises against submitting data with a start button", "https://design-system.service.gov.uk/components/button/#start-buttons").html_safe
```

Reply:
It's also probably worth adjusting the opening sentence above so it says something more like:

> Start buttons don't usually submit form data, so they use a link tag rather than a button tag.

Reply:
I've opened https://github.com/alphagov/govuk-design-system/pull/2238 to change the wording.